config.yaml

```diff
@@ -409,6 +409,305 @@ Images:
+  - 1.20.3-distroless
+  - 1.21.1
+  - 1.21.1-distroless
+  - 1.22.1
+  - 1.22.1-distroless
+  - 1.23.2
+  - 1.23.2-distroless
+  - 1.24.0
+  - 1.24.0-distroless
+  - 1.24.1
+  - 1.24.1-distroless
+  - 1.25.0
+  - 1.25.0-distroless
+  - 1.7.8
+  - 1.8.3
+  - 1.8.4
+  - 1.8.5
+  - 1.8.6
+  - 1.9.1
+  - 1.9.2
+  - 1.9.3
+  - 1.9.5
+  - 1.9.6
+  - 1.9.8
+- SourceImage: istio/kubectl
+  Tags:
+  - 1.5.10
+  TargetImageName: istio-kubectl
+- SourceImage: istio/kubectl
+  Tags:
+  - 1.5.9
+- SourceImage: istio/mixer
+  Tags:
+  - 1.7.0
+  - 1.7.1
+  - 1.7.3
+  - 1.7.6
+  TargetImageName: istio-mixer
+- SourceImage: istio/mixer
+  Tags:
+  - 1.5.9
+  - 1.7.8
+- SourceImage: istio/node-agent-k8s
+  Tags:
+  - 1.5.9
+- SourceImage: istio/pilot
+  Tags:
+  - 1.7.0
+  - 1.7.1
+  - 1.7.3
+  - 1.7.6
+  - 1.8.1
+  - 1.8.3
+  TargetImageName: istio-pilot
+- SourceImage: istio/pilot
+  Tags:
+  - 1.10.0
+  - 1.10.1
+  - 1.10.2
+  - 1.10.4
+  - 1.11.2
+  - 1.11.4
+  - 1.11.7
+  - 1.11.8
+  - 1.12.6
+  - 1.13.3
+  - 1.14.1
+  - 1.14.1-distroless
+  - 1.14.3
+  - 1.14.3-distroless
+  - 1.15.2
+  - 1.15.2-distroless
+  - 1.15.3
+  - 1.15.3-distroless
+  - 1.16.3
+  - 1.16.3-distroless
+  - 1.17.2
+  - 1.17.2-distroless
+  - 1.18.2
+  - 1.18.2-distroless
+  - 1.19.0
+  - 1.19.0-distroless
+  - 1.19.5
+  - 1.19.5-distroless
+  - 1.19.6
+  - 1.19.6-distroless
+  - 1.20.3
+  - 1.20.3-distroless
+  - 1.21.1
+  - 1.21.1-distroless
+  - 1.22.1
+  - 1.22.1-distroless
+  - 1.23.2
+  - 1.23.2-distroless
+  - 1.24.0
+  - 1.24.0-distroless
+  - 1.24.1
+  - 1.24.1-distroless
+  - 1.25.0
+  - 1.25.0-distroless
+  - 1.5.9
+  - 1.7.8
+  - 1.8.3
+  - 1.8.4
+  - 1.8.5
+  - 1.8.6
+  - 1.9.1
+  - 1.9.2
+  - 1.9.3
+  - 1.9.5
+  - 1.9.6
+  - 1.9.8
+- SourceImage: istio/proxyv2
+  Tags:
+  - 1.7.0
+  - 1.7.1
+  - 1.7.3
+  - 1.7.6
+  - 1.8.1
+  - 1.8.3
+  TargetImageName: istio-proxyv2
+- SourceImage: istio/proxyv2
+  Tags:
+  - 1.10.0
+  - 1.10.1
+  - 1.10.2
+  - 1.10.4
+  - 1.11.2
+  - 1.11.4
+  - 1.11.7
+  - 1.11.8
+  - 1.12.6
+  - 1.13.3
+  - 1.14.1
+  - 1.14.1-distroless
+  - 1.14.3
+  - 1.14.3-distroless
+  - 1.15.2
+  - 1.15.2-distroless
+  - 1.15.3
+  - 1.15.3-distroless
+  - 1.16.3
+  - 1.16.3-distroless
+  - 1.17.2
+  - 1.17.2-distroless
+  - 1.18.2
+  - 1.18.2-distroless
+  - 1.19.0
+  - 1.19.0-distroless
+  - 1.19.5
+  - 1.19.5-distroless
+  - 1.19.6
+  - 1.19.6-distroless
+  - 1.20.3
+  - 1.20.3-distroless
+  - 1.21.1
+  - 1.21.1-distroless
+  - 1.22.1
+  - 1.22.1-distroless
+  - 1.23.2
+  - 1.23.2-distroless
+  - 1.24.0
+  - 1.24.0-distroless
+  - 1.24.1
+  - 1.24.1-distroless
+  - 1.25.0
+  - 1.25.0-distroless
+  - 1.5.9
+  - 1.7.8
+  - 1.8.3
+  - 1.8.4
+  - 1.8.5
+  - 1.8.6
+  - 1.9.1
+  - 1.9.2
+  - 1.9.3
+  - 1.9.5
+  - 1.9.6
+  - 1.9.8
+- SourceImage: istio/sidecar_injector
+  Tags:
+  - 1.5.9
+- SourceImage: jaegertracing/all-in-one
+  Tags:
+  - 1.20.0
+  TargetImageName: jaegertracing-all-in-one
+- SourceImage: jaegertracing/all-in-one
+  Tags:
+  - "1.14"
```

Comment:
Why the quotes here?

Reply:
I think it's because it would be interpreted as a number without the quotes. See https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml:

> Use quotes to force a string, e.g. if your key or value is 10 but you want it to return a String and not a Fixnum, write '10' or "10".

I guess that the library I'm using for yaml decided they need to be there. Either way, this doesn't cause any issues.